src/components/Popup/Popup.js

```diff
@@ -197,74 +229,64 @@ class Popup extends PureComponent {
     }
   }
 
-  renderPopupHeader() {
-    const { t, title, showCloseButton, onCloseClick } = this.props;
-
-    return (
-      <div className="rs-popup-header">
-        {t(title)}
-        {showCloseButton ? (
-          <Button
-            className="rs-popup-close-bt"
-            title={`Popup ${t('Schliessen')}`}
-            onClick={() => onCloseClick()}
-          >
-            <MdClose focusable={false} />
-          </Button>
-        ) : null}
-      </div>
-    );
-  }
-
   render() {
     const {
       feature,
       showHeader,
       popupCoordinate,
-      className,
       children,
-      onKeyUp,
-      padding,
-      tabIndex,
+      title,
```

Comment:
Maybe rename `title` to header to avoid confusion with `titles`?

Reply:
yaes

Reply:
Done